src/api.rs

```diff
@@ -489,35 +491,42 @@ impl Config {
       first_pass_data: FirstPassData {
         frames: Vec::new(),
       },
+      pool,
+      },
+      config: self.enc.clone(),
```

Comment:
I had trouble reading this because of the indentation, could you please indent the inner struct contents?

Reply:
Sure